book/02-git-basics/sections/getting-a-repository.asc

```diff
@@ -7,7 +7,25 @@ The second clones an existing Git repository from another server.
 
 ==== Initializing a Repository in an Existing Directory
 
-If you're starting to track an existing project in Git, you need to go to the project's directory and type
+If you're starting to track an existing project in Git, you need to go to the project's directory
```

Comment:
If actually prefer if you made the change. That way you name is on all of
the commits for this contribution, and your name shows up in the blame
view. It's your improvement, you should get all the credit.
On Sun, May 31, 2015 at 9:59 AM Petar Enchev <notifications@github.com>
wrote:

> In book/02-git-basics/sections/getting-a-repository.asc
> <https://github.com/progit/progit2/pull/311#discussion_r31392165>:
>
> > @@ -7,7 +7,25 @@ The second clones an existing Git repository from another server.
> >
> >  ==== Initializing a Repository in an Existing Directory
> >
> > -If you're starting to track an existing project in Git, you need to go to the project's directory and type
> > +If you're starting to track an existing project in Git, you need to go to the project's directory
>
> I totally agree with you. Your way of explanation is much better than what
> I did (just to add the examples). What do you prefer:
> 1. Me to change the pull request with additional commit?
> 2. You to create an additional commit after you merge the PR?
> 3. Anything else?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/progit/progit2/pull/311/files#r31392165>.
>